src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -227,10 +234,12 @@ private void visitReturn(DetailAST ast) {
         }
 
         /**
-         * Increase the number of return statements.
+         * Increase the number of return statements and set method return type.
```

Comment:
Please use `context` instead of `method`

Reply:
done